src/res/translation/translation_sk_SK.ts

```diff
@@ -91,7 +91,7 @@
     <message>
         <location filename="../../util.cpp" line="510"/>
         <source>Slovak</source>
-        <translation type="unfinished"></translation>
+        <translation>slovenčina</translation>
```

Comment:
Should this be capitalised? Slovenčina

Reply:
Depends, but in general language names in Slovak are not capitalized (see http://www.slovencina.vselico.com/velkepismena.html [Slovak])